ui/src/app/views/workspace-edit.tsx

```diff
@@ -381,8 +438,42 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         cloneUserRole: false,
         loading: false,
         showUnderservedPopulationDetails: false,
-        showStigmatizationDetails: false
+        showStigmatizationDetails: false,
+        diseaseList: [],
+        diseaseHover: [],
+        diseaseEntryState: 0,
       };
+      this.searchTermChangedEvent = fp.debounce(300, this.diseaseSearch);
+      this.DS_START = 1;
```

Comment:
if these are just enums i don't quite understand why they have to be defined with values?  could you just initialize them as 
`const diseaseSearchStates = {DS_START, DS_ACTIVE, DS_SUGGEST, DS_HOVER}`
?

Reply:
+1, we have a pretty strong pattern to define constants like this as a Typescript enum, rather than as member variables, e.g. https://github.com/all-of-us/workbench/blob/1edcbd0eeeacf2eb3e3264be662702a53057ee39/ui/src/app/views/workspace-edit.tsx#L323

Reply:
Fixed